docs/build/autogenerate.rst

```diff
@@ -488,6 +496,13 @@ then a basic check for type equivalence is run.
 .. versionadded:: 0.7.6 - added support for the ``compare_against_backend()``
    method.
 
+For a custom dialect, you could also specify ``impl.type_synonyms``. This
```

Comment:
**mike bayer** (zzzeek) wrote:

this is getting into people making their own "impls" which I think is out of scope for this document, so I would take this blurb out.